coreblocks/lsu/dummyLsu.py

```diff
@@ -217,6 +219,16 @@ def elaborate(self, platform):
         instr_is_load = Signal()
         m.d.comb += instr_is_load.eq(current_instr.exec_fn.op_type == OpType.LOAD)
 
+        addr = Signal(self.gen_params.isa.xlen)
+        m.d.comb += addr.eq(current_instr.s1_val + current_instr.imm)
+
+        pmas = Record(self.gen_params.get(PMALayouts).pma_attrs_layout)
+        m.d.comb += pma_checker.addr.eq(addr)
+        m.d.comb += pmas.eq(pma_checker.result)
```

Comment:
The attribute `pma_checker.result` is already a `Record`, you gain nothing by this. Consider just:

```suggestion
        m.d.comb += pma_checker.addr.eq(addr)
        pmas = pma_checker.result
```

Also, in case when creating a new Record is actually needed, check out `Record.like`.